pkg/varcontext/interpolation/funcs.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interpolation
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/hashicorp/hil/ast"
+	"github.com/spf13/cast"
+)
+
+var hilStandardLibrary = createStandardLibrary()
+
+// createStandardLibrary instantiates all the functions and associates them
+// to their names in a lookup table for our standard library.
+func createStandardLibrary() map[string]ast.Function {
```

Comment:
I feel we should create another doc file that describes more about the HIL interpolation and also lists these functions. (Since I'm assuming these functions are the only ones that can be evaluated)

Especially if customers are going to be using these down the road.

Reply:
Added a new doc with info about hil and the available functions. This can be filled in more once we add TF support.